make.bash

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
```

Comment:
nit: add the following bash options. With those options, bash stops the execution immediately after any command fails. It would make debugging easier.
```suggestion
set -euo pipefail
```

Reply:
在新的PR處理